model/fv_control.F90

```diff
@@ -250,6 +255,7 @@ subroutine fv_control_init(Atm, dt_atmos, this_grid, grids_on_this_pe, p_split)
      logical , pointer :: nudge_ic
      logical , pointer :: ncep_ic
      logical , pointer :: nggps_ic
+   logical , pointer :: hrrrv3_ic
```

Comment:
This isn't lined up and looks weird

Reply:
We can fix it in the master branch and bring it over.